spec.emu

```diff
@@ -828,7 +811,7 @@ contributors: Ujjwal Sharma, Younies Mahmoud
           <li>[[MicrosecondsDisplay]] is one of the String values *"auto"* or *"always"* identifying when to display the microseconds field.</li>
           <li>[[NanosecondsStyle]] is one of the String values *"long"*, *"short"*, *"narrow"*, or *"numeric"* identifying the formatting style used for the nanoseconds field.</li>
           <li>[[NanosecondsDisplay]] is one of the String values *"auto"* or *"always"* identifying when to display the nanoseconds field.</li>
-          <li>[[FractionalDigits]] is a Number value, identifying the number of fractional digits to be used with numeric styles, or is *undefined*.</li>
+          <li>[[FractionalDigits]] is a non-negative integer, identifying the number of fractional digits to be used with numeric styles, or is *undefined*.</li>
```

Comment:
In the Temporal proposal, we will make extensive use of storing integers in internal slots.

Reply:
> I've done a cursory look through 262, and I can't find any prior art for internal slots storing integers (rather than integer Numbers) anywhere there [...]

The [[ByteLength]], [[ByteOffset]], and [[ArrayLength]] internal slots of TypedArray objects are all mathematical values instead of Number values. Same applies for [[ArrayBufferByteLength]] in (Shared)ArrayBuffer objects, and [[ByteLength]] and [[ByteOffset]] in DataView objects.

Reply:
oh, duh, of course. Thank you, Philip & Anba. This one looks good to me!